clients/client-java/src/main/java/com/datastrato/gravitino/client/GravitinoClientBase.java

```diff
@@ -218,6 +218,17 @@ public Builder<T> withSimpleAuth() {
       return this;
     }
 
+    /**
+     * Sets the simple mode authentication for Gravitino with a specific username
+     *
+     * @param userName The username of the user.
+     * @return This Builder instance for method chaining.
+     */
+    public Builder<T> withSimpleAuth(String userName) {
+      this.authDataProvider = new SimpleTokenProvider(userName);
```

Comment:
Could you check the userName in the method `build`?

Reply:
the 'build' function has oauth and kerberos AuthDataProvider. so I think we should add the check in the withSimpleAuth like this.
is it ok?

Reply:
For builder pattern, we usually check them in the build method.